modules/ROOT/pages/setup.adoc

```diff
@@ -51,17 +57,12 @@ Both roles have unrestricted access, but one role works in a Production environm
 * Partner Manager Admin (Sandbox) - Can perform all operations across Partner Manager for all Sandbox environments.
 
 . Verify that you have access to Partner Manager.
-This means that you must have the required entitlements and have assigned the `Partner Manager Administrator` role in Access Management for the environment.
+This means that you must have the required entitlements and have assigned the `Partner Manager Admin` role in Access Management for the environment.
```

Comment:
What is an "entitlement"? Could this simply be You must have the Partner Manager Admin role for access management?

GLOBAL: Do not capitalize access management.

Official name | access management | As a capability, this is a descriptor of the functionality. These are predetermined names for capabilities, so while they are not a name, they should be described consistently
-- | -- | --